### ISSUE TITLE
'Cannot find module './parser' from cssx-transpiler/lib/cssx-transpiler: 149

### ISSUE THREAD

**Issue by shedali** [krasimir/cssx#1]
I receive this error when trying to require cssx-transpiler.

**Comment by krasimir** [comment]
Thanks for reporting this @shedali. Just to clarify, you installed `cssx-transpiler` and try using `require('cssx-transpiler')`.

**Comment by shedali** [comment]
Yes that is right

**Comment by krasimir** [comment]
Ok, thanks. It will probably fixed tomorrow early morning.

**Comment by shedali** [comment]
great! Look forward to trying it out

**Comment by krasimir** [comment]
I just tried reproducing the problem but I wasn't able. Here is my setup:

`index.js` file:
```js
var code = require('fs').readFileSync('./code.js').toString('utf8');
var cssxTranspiler = require('cssx-transpiler');
var transpiled = cssxTranspiler(code, { minified: false });
console.log(transpiled);
```
`code.js` file:
```js
var styles = function (margin) {
  <style>
    body {
      margin: `margin`px;
      padding: 0;
    }
  </style>
};
```
I run `npm i cssx-transpiler` and then `node ./index.js` and the result is:
![image](https://cloud.githubusercontent.com/assets/528677/13836628/f8db3238-ec0b-11e5-8ab8-847bdc43bea8.png)

I tried under mac (node version v0.12.0) and windows (node version v4.2.0). In both cases I got the same result.

In general, if you use `cssx-transpiler` package you get just one compiled file so there is no `./parser` there. Are you trying to use this repo directly?

P.S.
Here is how the file structure looks like:
![image](https://cloud.githubusercontent.com/assets/528677/13836676/6f16ad24-ec0c-11e5-8f1e-4edd880edb8d.png)
